duckduckgo/bloom_cpp
# Add GitHub Actions support

Task/Issue URL: https://app.asana.com/0/903182762228930/903861304087285/f
Tech Design URL: 
CC: 

**Description**:

This PR adds GitHub Actions support, running on Linux, macOS, and Windows.

This also fixes test suite compilation on M1 Macs, as there was a hardcoded path to the Intel Homebrew install location.

**Steps to test this PR**:
1. Check out this branch and run `./run_test.sh`
1. Verify that a failed test on this branch is surfaced correctly in the PR status report


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
